pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -466,3 +466,167 @@ func TestRestartCountByLogDir(t *testing.T) {
 		assert.Equal(t, count, tc.restartCount, "count %v should equal restartCount %v", count, tc.restartCount)
 	}
 }
+
+func TestKillContainerGracePeriod(t *testing.T) {
```

Comment:
nit: should we name this closer to the function we are testing ` setTerminationGracePeriod` ? Like TestSetTerminationGracePeriod or TestTerminationGracePeriod ?